### PR TITLE
[Feature #109714504] Enable commenting on projects

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: public/js/app.js [hunk 1]
```diff
@@ -19,6 +19,61 @@ jQuery( document ).ready(function( $ ){
...
+        }
+    )
+    .error(
+      function(respData)
+      {
+        console.log('An error occurred');
+      }
+    );
+  }
+}
```
Comment (unicodeveloper) [inline public/js/app.js:65]: Please fix all houndCI issues and remove every `console.log` statements.

File: app/Http/Controllers/CommentController.php [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+class CommentController extends Controller
+{
+    /**
+     * Saves project comment and returns JSON data
+     */
+    public function makeComment($id, CommentRequest $request, PostComment $comment)
+    {
+        $user = Auth::user();
+
+        return Response::json($comment->saveProjectComment($id, $user, $request));
```
Comment (unicodeveloper) [inline app/Http/Controllers/CommentController.php]: **Code Quality - Critical**

Consistency is key!. Use the `response()` helper function here just like you did in the destroy method

File: app/Helpers/PostComment.php [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+    public function __construct(Comment $comment)
+    {
+        $this->comment = $comment;
+    }
+
+    /**
+     *
+     */
+    public function saveProjectComment($id, User $user, CommentRequest $request)
```
Comment (unicodeveloper) [inline app/Helpers/PostComment.php:22]: Why do we have this empty comment?

File: app/Helpers/PostComment.php [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+        $this->comment->comment    = $request->comment;
+        $this->comment->user_id    = $user->id;
+        $this->comment->project_id = $id;
+
+        $this->comment->save();
+
+        $commentId   = Comment::where('comment', $request->comment)->first()->id;
+        $commentRepo = new CommentRepository;
+
+        return ['avatar'    => $user->avatar,
```
Comment (unicodeveloper) [inline app/Helpers/PostComment.php]: **Code Quality - Nitpick**

`'avatar' => $user->avatar` should be on a new line, not on thesame line as the opening `[`

File: app/Http/Controllers/CommentController.php (hunk 2 above)
Comment (ghost) [inline app/Http/Controllers/CommentController.php]: What does it mean by "Bad line breaking before '+'."?

Comment (ghost) [inline app/Http/Controllers/CommentController.php]: @unicodeveloper ^^^

Comment (unicodeveloper) [inline app/Http/Controllers/CommentController.php]: It means you don't need that line break. The times you actually need to use that `+` should be when you are concatenating the response data to the HTML elements.

Here you used it almost 100 times

**Pull request merged**